frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -596,21 +595,24 @@ export const BillingProduct = ({ product }: { product: BillingProductV2Type }):
                         </div>
                     )}
                 </div>
-                {(showUpgradeCTA || (isOnboarding && !product.contact_support)) && (
+                {(upgradePlan ||
+                    (!upgradePlan && !product.current_amount_usd) ||
+                    (isOnboarding && !product.contact_support)) && (
                     <div
                         data-attr={`upgrade-card-${product.type}`}
                         className={`border-t border-border p-8 flex justify-between ${
-                            product.subscribed ? 'bg-success-highlight' : 'bg-warning-highlight'
+                            !upgradePlan ? 'bg-success-highlight' : 'bg-warning-highlight'
                         }`}
                     >
                         <div>
-                            <h4 className={`${product.subscribed ? 'text-success-dark' : 'text-warning-dark'}`}>
-                                You're on the {product.subscribed ? 'paid' : 'free'} plan for {product.name}.
+                            <h4 className={`${!upgradePlan ? 'text-success-dark' : 'text-warning-dark'}`}>
```

Comment:
Wonder if we could make this more clear by doing something like the following: 
<img width="1211" alt="Screenshot 2024-02-28 at 6 37 37 PM" src="https://github.com/PostHog/posthog/assets/21014901/06143fb5-e888-4b34-8ccf-38b7268e6890">